README.md

```diff
@@ -1,101 +1,134 @@
-# Hotel System
-
-A friend of ours in an heiress and she seeks our assistance in creating a hotel management system. This system will be used primarily by employees working at the front desk as people
-
+# Hotel
 
 ## Learning Goals
+
 Reinforce and practice all of the Ruby and programming concepts we've covered in class so far:
 - Design a system using object-oriented principles
 - Create and instantiate classes with attributes
 - Create class and instance methods within our classes
 - Write pseudocode and create tests to drive the creation of our code
 
-## Expectations
-This project will have requirements that are more open-ended than previous projects you have worked on. This is expected. As this is your final Ruby project, you will be expected to:
+This is a [level 3](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md), individual project.
```

Comment:
not a big deal, but we should standardize the language on this (the readme calls these "stages," not levels)

Reply:
Meant to change this and forgot. Good eye.